code/src/index.css

```diff
@@ -5,9 +5,96 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(230, 223, 223);
+  background-image: url("./assets/hero-image.jpg");
+  background-size: cover;
```

Comment:
To make the image to be covered in the whole page you need to add a height `height: 100vh;`